libs/dh/reports/feature-measurements-reports/src/components/request-report/request-report-modal.component.ts

```diff
@@ -160,8 +185,40 @@ export class DhRequestReportModal extends WattTypedModal<MeasurementsReportReque
     this.form.controls.period.updateValueAndValidity();
   });
 
+  private switchToMeteringPointIDsEffect = effect(() => {
+    const switchToMeteringPointIDs = this.switchToMeteringPointIDsChanges();
+
+    if (switchToMeteringPointIDs) {
+      this.form.controls.meteringPointTypes.reset();
```

Comment:
Nitpick: reset can be moved out of if else

Reply:
The `reset` in the `else` case resets `meteringPointIDs` controller. This one resets the `meteringPointTypes` :)

Reply:
Still have not had my coffee.. :-)